src/app/components/event/report/EventReportDrawer.vue

```diff
@@ -89,6 +105,24 @@
           </ButtonColored>
         </DrawerClose>
       </AppStep>
+      <AppStep v-slot="attributes" :is-active="step === 'error'">
+        <ButtonColored
+          v-bind="attributes"
+          :aria-label="t('backToReport')"
+          variant="primary-critical"
+          @click="restart"
+        >
+          {{ t('backToReport') }}
+        </ButtonColored>
+        <DrawerClose v-bind="attributes" as-child>
+          <ButtonColored
+            :aria-label="t('buttonReportCancel')"
+            variant="secondary-critical"
+          >
+            {{ t('buttonReportCancel') }}
+          </ButtonColored>
+        </DrawerClose>
```

Comment:
Stepper should restart when cancel is clicked here so that the error does not show again when reopening the drawer

Reply:
done, you can resolve once checked